media-proxy/src/mesh/conn.cc

```diff
@@ -208,6 +211,11 @@ Result Connection::set_link(context::Context& ctx, Connection *new_link)
     return set_result(Result::success); // TODO: return error if needed
 }
 
+Connection * Connection::link()
```

Comment:
```suggestion
Connection * Connection::get_link()
```
I think getter is more readable when it has `get_` prefix

Reply:
Following your suggestion, we'll have to change all similar method names:
```cpp
Kind kind();
State state();
Status status();
Connection * link();
```
this way:
```cpp
Kind get_kind();
State get_state();
Status get_status();
Connection * get_link();
```
Personally, I don't like it. It seems like a C# or Java approach to name getters/setters. I like the `state() / set_state()` way of giving names much more because it's less words, and it's the Go style :-)